src/main/scala/ShowResults.scala

```diff
@@ -0,0 +1,134 @@
+import com.github.tototoshi.csv.CSVWriter
+import scala.concurrent.{Await, ExecutionContext, Future}
+import com.typesafe.scalalogging.LazyLogging
+import scala.collection.JavaConverters._
+import scala.collection.immutable.HashMap
+import scala.collection.mutable.{ArrayBuffer, ListBuffer}
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.io._
+import scala.util._
+import CsvWriter.{writeCountriesWithMaxGasPlants, writeCountriesWithMinGasPlants, writeHeaderForPowerPlants, writePlantsInEachContinent, writeTotalPower, writeYearWithMaxNumOfPlants, outputFilePowerPlants, outputFileGeoStat}
+
+import java.nio.file.{Files, FileSystems}
+
+case class PlantsData(country_code: String, country_long: String, capacity_MV: Double, commissioning_year: String, fuels: Array[String])
```

Comment:
Names of fields does not match naming convention. Check out this article once again - https://github.com/databricks/scala-style-guide